xml/art-nvidia-vgpu.xml

```diff
@@ -0,0 +1,1137 @@
+      VT-D/IOMMU and SR-IOV technologies, and that they are enabled in BIOS.
+     </para>
+    </step>
+    <step>
+     <para>
+      Enable IOMMU. Verify that it is included in the boot command line:
+     </para>
+<screen>
+cat /proc/cmdline
+BOOT_IMAGE=/boot/vmlinuz-default [...] intel_iommu=on [...]
+</screen>
+     <para>
+      If not, add the following line to <filename>/etc/default/grub</filename>.
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        For Intel CPUs:
+       </para>
+<screen>GRUB_CMDLINE_LINUX="intel_iommu=on"</screen>
+       <para>
+        For AMD CPUs:
+       </para>
+<screen>GRUB_CMDLINE_LINUX="amd_iommu=on"</screen>
+      </listitem>
+     </itemizedlist>
+     <para>
+      Then generate new &grub; configuration file and reboot:
+     </para>
+<screen>
+&prompt.sudo;grub2-mkconfig -o /boot/grub2/grub.cfg
+&prompt.sudo;systemctl reboot
+</screen>
+     <tip>
+      <para>
+       You can verify that IOMMU is loaded by running the following command:
+      </para>
+<screen>sudo dmesg | grep -e IOMMU</screen>
+     </tip>
+    </step>
+    <step>
+     <!-- 2021-09-21 tbazant: this seems out of place here:
+      "After finishing the driver installation, run
+      /usr/lib/nvidia/sriov-manage -e slot:bus:domain.function"
+      -->
+     <para>
+      Enable SR-IOV. Refer to
+      <link
+      xlink:href="https://documentation.suse.com/sles/15-SP3/single-html/SLES-virtualization/#sec-libvirt-config-io"/>
+      for useful information.
+     </para>
+    </step>
+    <step>
+     <para>
+      Disable the nouveau kernel module by adding the following line it to the
+      top of the <filename>/etc/modprobe.d/50-blacklist.conf</filename> file:
+     </para>
+<screen>blacklist nouveau</screen>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="configure-nvidia-vgpu-kvm-driver">
+   <title>Install the &nvidia; &kvm; driver</title>
+   <procedure>
+    <step>
+     <para>
+      Exit from the graphical mode:
+     </para>
+<screen>&prompt.sudo;init 3</screen>
+    </step>
+    <step>
+     <para>
+      Install the <package>kernel-default-devel</package> package and its
+      dependencies:
+     </para>
+<screen>&prompt.sudo;zypper in kernel-default-devel</screen>
+    </step>
+    <step>
+     <para>
+      Download the vGPU software from the &nvidia; portal, make it executable,
+      and run it:
+     </para>
+<screen>
+&prompt.user;chmod +x NVIDIA-Linux-x86_64-450.55-vgpu-kvm.run
+&prompt.sudo;./NVIDIA-Linux-x86_64-450.55-vgpu-kvm.run
+</screen>
+     <tip>
+      <para>
+       To enable dynamic kernel module support in order to get the module
+       rebuilt automatically when new a new kernel is installed, add the
+       <option>--dkms</option> option:
+      </para>
+<screen>&prompt.sudo;./NVIDIA-Linux-x86_64-450.55-vgpu-kvm.run --dkms</screen>
+     </tip>
+    </step>
+    <step>
+     <para>
+      When the driver installation is finished, reboot the system:
```

Comment:
Add a note where to find the log of the installation: /var/log/nvidia-installer.log

Reply:
I have had a lot of problems with nVidia drivers myself. The above is indeed useful information. What I cannot remember (or check now, as I am on an Intel-only system) is whether the cited log file is the main one, or whether it merely is a pointer to the "real" log file located elsewhere.

If it is the latter, it might also be useful to point to where the "real" log file is located.

Another issue that I found is that, if I remember correctly, the nVidia drivers also need the `dkms` package installed, and it in turn needs kernel-headers and the core build tools (`zypper in -t pattern devel_basis`). @aginies, do you think this is worth mentioning too, or does it not apply here?